src/components/LifeEventsListView.vue

```diff
@@ -118,6 +158,26 @@ export default {
             if (this.includeUpcomingEvents && !isFuture(e.redZone.start)) return true
             if (! this.includeUpcomingEvents && !isFuture(e.redZone.start)) return true
           })
+          .filter(e => {
+            if(this.filterTag === undefined) return true
+            console.log(this.filterTag)
+            if(e.title.includes("#"+this.filterTag)) return true
+            return false
+          })
+    },
+    listOfCurrentTags() {
+      let tags = []
+      this.events.forEach(element => {
+        let titleArray = element.title.split("#")
+          titleArray.forEach((tag, index) => {
+            if(index != 0){
```

Comment:
Momentan sind per Definition nur Tags am Ende des Titels erlaubt. **Das ist für die erste Version vollkommen in Ordnung.**

Ich schlage vor ein neues Issue aufzumachen und es nochmal genauer zu spezifieren. Im Grunde fände ich ein Tagging à la Twitter nice:

1. Arzttermin vereinbaren #arzt #health
2. #Arzt|termin vereinbaren #health
3. #Arzt anrufen wegen #Termin

Das ist massiv komplizierter und sicherlich ein neuer PR!

Reply:
Issue angelegt :-)